intro/scipy/index.rst

```diff
@@ -751,88 +751,116 @@ whether the means of two sets of observations are significantly different::
-First the function computing the derivative of the position needs to be defined::
+and plot the result::
 
-    >>> def calc_derivative(time, ypos):
-    ...     return -2 * ypos
+    >>> plt.plot(res.t, res.y[0])  # doctest: +SKIP
+    >>> plt.xlabel('t')  # doctest: +SKIP
+    >>> plt.ylabel('y')  # doctest: +SKIP
+    >>> plt.title('Solution of Initial Value Problem')  # doctest: +SKIP
 
 .. image:: auto_examples/images/sphx_glr_plot_solve_ivp_simple_001.png
     :target: auto_examples/plot_solve_ivp_simple.html
     :scale: 70
     :align: right
 
+Let us integrate a more complex ODE: a `damped
+spring-mass oscillator
+<https://en.wikipedia.org/wiki/Harmonic_oscillator#Damped_harmonic_oscillator>`__.
+The position of a mass attached to a spring obeys the 2nd order ODE
+:math:`\ddot{y} + 2 \zeta \omega_0  \dot{y} + \omega_0^2 y = 0` with natural frequency
+:math:`\omega_0 = \sqrt{k/m}`, damping ratio :math:`\zeta = c/(2 m \omega_0)`,
+spring constant :math:`k`, mass :math:`m`, and damping coefficient :math:`c`.
 
-Then, to compute ``y`` as a function of time::
-
-    >>> solution = sp.integrate.solve_ivp(calc_derivative, (0, 4), y0=(1,))
+Before using :func:`scipy.integrate.solve_ivp`, the 2nd order ODE
+needs to be transformed into a system of first-order ODEs. Note that
 
-.. raw:: html
+.. math::
 
-   <div style="clear: both"></div>
+    \begin{eqnarray}
+    \frac{dy}{dt} &=& \dot{y} \\
+    \frac{d\dot{y}}{dt} &=& \ddot{y} = -(2 \zeta \omega_0  \dot{y} + \omega_0^2 y)
+    \end{eqnarray}
 
-Let us integrate a more complex ODE: a `damped
-spring-mass oscillator
-<https://en.wikipedia.org/wiki/Harmonic_oscillator#Damped_harmonic_oscillator>`__.
-The position of a mass attached to a spring obeys the 2nd order *ODE*
-:math:`y'' + 2 \varepsilon \omega_0  y' + \omega_0^2 y = 0` with
-:math:`\omega_0^2 = k/m` with :math:`k` the spring constant, :math:`m` the mass
-and :math:`\varepsilon = c/(2 m \omega_0)` with :math:`c` the damping coefficient. We set::
+If we define :math:`z = [z_0, z_1]` where :math:`z_0 = y` and :math:`z_1 = \dot{y}`,
+then the first order equation:
 
-    >>> mass = 0.5  # kg
-    >>> kspring = 4  # N/m
-    >>> cviscous = 0.4  # N s/m
+.. math::
 
-Hence::
+    \usepackage{amsmath}
+    \begin{eqnarray}
+    \frac{dz}{dt} =
+    \begin{bmatrix}
+        \frac{dz_0}{dt} \\
+        \frac{dz_1}{dt}
+    \end{bmatrix} =
+    \begin{bmatrix}
+        z_1  \\
+        -(2 \zeta \omega_0  z_1 + \omega_0^2 z_0)
+    \end{bmatrix}
+    \end{eqnarray}
 
-    >>> eps = cviscous / (2 * mass * np.sqrt(kspring/mass))
-    >>> omega = np.sqrt(kspring / mass)
+is equivalent to the original second order equation.
 
-The system is underdamped, as::
+We set::
 
-    >>> eps < 1
-    True
+    >>> m = 0.5  # kg
+    >>> k = 4  # N/m
+    >>> c = 0.4  # N s/m
+    >>> zeta = c / (2 * m * np.sqrt(k/m))
+    >>> omega = np.sqrt(k / m)
 
-For :func:`~scipy.integrate.solve_ivp`, the 2nd order equation
```

Comment:
This is often tricky for those who are unfamiliar with the process. I tried to make it more explicit.